protoc-gen-twirp/generator.go

```diff
@@ -1104,8 +1111,13 @@ func (t *twirp) generateServerRouting(servStruct string, file *descriptor.FileDe
 	t.P(`  switch req.URL.Path {`)
 	for _, method := range service.Method {
 		path := pathFor(file, service, method)
+		pathCamel := pathForCamelCompat(file, service, method)
 		methName := "serve" + stringutils.CamelCase(method.GetName())
-		t.P(`  case `, strconv.Quote(path), `:`)
+		caseCondition := strconv.Quote(path)
```

Comment:
Could you add a test in `internal/twirptest` that verifies this new behavior? You will also need to run `make` to re-generate the code with these changes